app/models/user.rb

```diff
@@ -133,10 +131,8 @@ def email_bounced?
   end
 
   def login_token_expiration_time
-    (
-      login_token_generated_at +
-        Rails.application.secrets.login_token_time_limit
-    ).strftime('%B %-d, %Y at %l:%M %p')
+    (login_token_generated_at +
+     Rails.application.secrets.login_token_time_limit).strftime('%B %-d, %Y at %l:%M %p')
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)